scripts/extensions/sams/src/components/assets/assetListPanel.tsx

```diff
@@ -35,24 +70,64 @@ export function getAssetListPanel(superdesk: ISuperdesk) {
                 );
             } else if (this.props.listStyle === ASSET_LIST_STYLE.GRID) {
                 return (
-                    <GridList className="sd-margin--1">
-                        {this.props.assets.map((asset) => (
-                            <AssetGridItem key={asset._id} asset={asset} />
-                        ))}
-                    </GridList>
+                    <PageLayout
```

Comment:
The `<PageLayout>` component is designed to be used higher up in the hierarchy of components. In this use case it is defined in the `SamsWorkspaceComponent`. This will also enable this component to focus solely on listing of the Assets.

Reply:
Have moved assetPreviewPanel to SamsWorkspaceComponent.